java/src/nodejt400/JdbcJsonClient.java

```diff
@@ -302,6 +309,10 @@ else if (value == null)
 			{
 				throw e;
 			}
+			catch (IllegalArgumentException e)
```

Comment:
Why are you catching this exception in order to re-throw it. Without doing anything else with it?

Seems like un-necessary code to me.

Reply:
I added that beacuse you catched the SQLException but the base64 decode can throw an IllegalArgumentException. You also retrow the  SQLException and catch it everytime setParams function is called so I did too in roder to maintain the same behaviour.

Reply:
and since java compiler is 1.6+ I could not use the catch(SQLException | IllegalArgumentException e) notation.